client/pages/reading-list.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import Header from '../components/header';
+import Loader from '../components/loader';
 
 export default class ReadingList extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       isBuyClicked: false,
+      isLoading: false,
```

Comment:
Instead of setting `isLoading` to `false` and then immediately setting it to `true` in `componentDidMount`, just start it as `true`.